scipy/sparse/linalg/_eigen/tests/test_svds.py

```diff
@@ -324,6 +324,8 @@ def err(tol):
             assert error > accuracy/10
 
     def test_svd_v0(self):
+        if self.solver == 'lobpcg':
+            pytest.skip("LOBPCG solutions may be not the same")
```

Comment:
What changed here that requires this skip (which wasn't previously necessary, apparently)?

Reply:
I no longer remember, so trying to run it again now